### PR TITLE
Main section

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
here you can find some info about the structures of the project
the project was build using HTML and designed by CSS flex
the sections are: main header section including the logo and the nave bar
and the main section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Aladdin</title>
+</head>
+<body>
+    <header>
+        <nav class="nav">
+            <ul>
+                <li id="logo"><a href="#">Aladdin Alsalem</a></li>
+                <li id="nav-btn"><a href="/"><i class="fas fa-bars hamburger-icon menu"></i></a></li>
```
Comment (SajjadAhmad14) [inline index.html:23]: The hamburger button should have a nav tag. Please use the nav tag only for the hamburger menu.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        </nav>
+    </header>
+
+    <main>
+        <section class="main">
+            <div class="head">
+                <h1>Hey There <br>I'm Aladdin</h1>
+                <h4>I am a Software Developer</h4>
+                <p>I can help you build a product, feature or website. Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, do not hesitate to contact me.</p>
```
Comment (SajjadAhmad14) [inline index.html:34]: Please use font color and font family mentioned in the design.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                <li id="logo"><a href="#">Aladdin Alsalem</a></li>
+                <li id="nav-btn"><a href="/"><i class="fas fa-bars hamburger-icon menu"></i></a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <section class="main">
+            <div class="head">
+                <h1>Hey There <br>I'm Aladdin</h1>
```
Comment (SajjadAhmad14) [inline index.html:31]: There is a bit large margin between heading and paragraph. Please see the below screenshot and fix the spacing.
intro

File: README.md [hunk 4]
```diff
@@ -1 +1,44 @@
 "# Aladdin_portfolio" 
+    
+# Hello Microverse
```
Comment (SajjadAhmad14) [inline README.md:3]: Please use [this](https://github.com/microverseinc/readme-template) template and customize it for your own need. README should have a project name and details.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Aladdin Alsalem, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [ ] GitHub flow is used properly.
- [ ] Linter checks are passing in PR.
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello, @AlaaAlsalem  great job working on the requested changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

- [x] [OPTIONAL] Well done on using the Microverse Readme template. To make it more descriptive, you can add a screenshot to give people a glimpse of how your project looks.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlaaAlsalem** [comment]
merge after review

**Pull request merged**